dao.py

```diff
@@ -133,7 +133,8 @@ def update_pending_tournament(self, pending_tournament):
         if len(pending_tournament.raw) == 0:
             raise UpdateTournamentException("Can't update a pending tournament with an empty 'raw' field because it will be overwritten!")
 
-        return self.pending_tournaments_col.update({'_id': pending_tournament.id}, pending_tournament.get_json_dict())
```

Comment:
No i didnt. Wtf. 

Sent from my iPhone

> On Apr 16, 2015, at 1:37 AM, garsh0p <notifications@github.com> wrote:
> 
> In dao.py:
> 
> > @@ -133,7 +133,8 @@ def update_pending_tournament(self, pending_tournament):
> >          if len(pending_tournament.raw) == 0:
> >              raise UpdateTournamentException("Can't update a pending tournament with an empty 'raw' field because it will be overwritten!")
> >  
> > -        return self.pending_tournaments_col.update({'_id': pending_tournament.id}, pending_tournament.get_json_dict())
> I don't think you meant to delete this line did you?
> 
> —
> Reply to this email directly or view it on GitHub.
>